baybe/utils/basic.py

```diff
@@ -16,6 +17,23 @@
 _U = TypeVar("_U")
 
 
+class UnspecifiedType(enum.Enum):
```

Comment:
This class is somehow causing errors in the doc building pipeline:

`WARNING: py:obj reference target not found: baybe.utils.basic.UnspecifiedType.UNSPECIFIED [ref.obj]`

Reply:
Since we still haven't understood the real mechanics of the problem, the best I could do is to set an explicit ignore (like for other cases we already encountered). If you have a better suggestion, feel free to reopen